From 61c487e6a1985e52307d6df5834b610a50219819 Mon Sep 17 00:00:00 2001
From: Phil Elwell <phil@raspberrypi.org>
Date: Wed, 22 May 2019 13:29:56 +0100
Subject: [PATCH] overlays: Update upstream overlay

The recent DT/overlay changes have had a corresponding effect on the
upstream overlay, which is a composite of the vc4-kms-v3d and dwc2
overlays.

Signed-off-by: Phil Elwell <phil@raspberrypi.org>
---
 .../boot/dts/overlays/upstream-overlay.dts    | 41 ++-----------------
 1 file changed, 3 insertions(+), 38 deletions(-)

--- a/arch/arm/boot/dts/overlays/upstream-overlay.dts
+++ b/arch/arm/boot/dts/overlays/upstream-overlay.dts
@@ -52,42 +52,36 @@
 	fragment@7 {
 		target = <&pixelvalve0>;
 		__overlay__ {
-			interrupts = <2 13>;
 			status = "okay";
 		};
 	};
 	fragment@8 {
 		target = <&pixelvalve1>;
 		__overlay__ {
-			interrupts = <2 14>;
 			status = "okay";
 		};
 	};
 	fragment@9 {
 		target = <&pixelvalve2>;
 		__overlay__ {
-			interrupts = <2 10>;
 			status = "okay";
 		};
 	};
 	fragment@10 {
 		target = <&hvs>;
 		__overlay__ {
-			interrupts = <2 1>;
 			status = "okay";
 		};
 	};
 	fragment@11 {
 		target = <&hdmi>;
 		__overlay__ {
-			interrupts = <2 8>, <2 9>;
 			status = "okay";
 		};
 	};
 	fragment@12 {
 		target = <&v3d>;
 		__overlay__ {
-			interrupts = <1 10>;
 			status = "okay";
 		};
 	};
@@ -98,37 +92,29 @@
 		};
 	};
 	fragment@14 {
-		target-path = "/soc/dma";
-		__overlay__ {
-			brcm,dma-channel-mask = <0x7f35>;
-		};
-	};
-	fragment@15 {
 		target = <&clocks>;
 		__overlay__ {
 			claim-clocks = <BCM2835_PLLD_DSI0 BCM2835_PLLD_DSI1 BCM2835_PLLH_AUX BCM2835_PLLH_PIX>;
 		};
 	};
-	fragment@16 {
+	fragment@15 {
 		target = <&vec>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@17 {
+	fragment@16 {
 		target = <&txp>;
 		__overlay__ {
 			status = "okay";
 		};
 	};
-	fragment@18 {
+	fragment@17 {
 		target = <&usb>;
 		#address-cells = <1>;
 		#size-cells = <1>;
 		dwc2_usb: __overlay__ {
 			compatible = "brcm,bcm2835-usb";
-			reg = <0x7e980000 0x10000>;
-			interrupts = <1 9>;
 			dr_mode = "otg";
 			g-np-tx-fifo-size = <32>;
 			g-rx-fifo-size = <256>;
@@ -136,25 +122,4 @@
 			status = "okay";
 		};
 	};
-	fragment@19 {
-		target = <&uart1>;
-		__overlay__ {
-			interrupt-parent = <&intc>;
-			interrupts = <0x1 0x1d>;
-		};
-	};
-	fragment@20 {
-		target = <&spi1>;
-		__overlay__ {
-			interrupt-parent = <&intc>;
-			interrupts = <0x1 0x1d>;
-		};
-	};
-	fragment@21 {
-		target = <&spi2>;
-		__overlay__ {
-			interrupt-parent = <&intc>;
-			interrupts = <0x1 0x1d>;
-		};
-	};
 };
